jul-ecs-formatter/pom.xml

```diff
@@ -27,6 +27,17 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.30</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-jdk14</artifactId>
```

Comment:
Isn't this dependency redundant? Can it be removed?

Reply:
I just checked. The slf4j-jdk14 dependency is necessary for the mdc test to work. Without this dependency Mdc.put does not do anything so then the test fails (Without a slf4j implementation there is no StaticMDCBinder which slf4j needs). 

I can make it a scope test and it should still work. The slf4j api dependency is needed because I call methods on it from the library.

Reply:
Ah, got it. Test scope sounds like the way to go then.

Reply:
I've added it